packages/mask/src/plugins/Avatar/utils/index.ts

```diff
@@ -91,3 +91,8 @@ export function formatText(name: string, tokenId: string) {
     }
     return `${_name} #${token}`
 }
+
+export function formatTokenId(symbol: string, tokenId: string) {
+    const name = `${symbol} #${tokenId}`
+    return name.length > 18 ? name.slice(0, 18) + '...' : name
+}
```

Comment:
Use this existed util function?
https://github.com/DimensionDev/Maskbook/blob/a9e636eab7fb68a5532966de569c93c6c882788c/packages/web3-shared/evm/utils/formatter.ts#L61-L64

Reply:
different needs